From ba8ae56d6836307cc7d36698173d3f2bcae12984 Mon Sep 17 00:00:00 2001
From: David Pacheco <dap@joyent.com>
Date: Mon, 14 Nov 2016 15:22:43 -0800
Subject: [PATCH] TOOLS-1613 cr.joyent.us needs to work around gerrit issue
 3885 Reviewed by: Alex Wilson <alex.wilson@joyent.com> Approved by: Alex
 Wilson <alex.wilson@joyent.com>

---
 images/appserver/etc/gerrit.config.base | 1 +
 1 file changed, 1 insertion(+)

diff --git a/images/appserver/etc/gerrit.config.base b/images/appserver/etc/gerrit.config.base
index ef22120..79b5d2f 100644
--- a/images/appserver/etc/gerrit.config.base
+++ b/images/appserver/etc/gerrit.config.base
@@ -16,6 +16,7 @@
 
 [index]
 	type = LUCENE
+	threads = 1
 
 [auth]
 	type = OAUTH
-- 
2.21.0

